test/hexpm_web/controllers/web_auth_controller.exs

```diff
@@ -0,0 +1,85 @@
+defmodule HexpmWeb.WebAuthControllerTest do
```

Comment:
This file needs to be named `web_auth_controller_test.exs`, otherwise tests won't run.